src/azure-cli/HISTORY.rst

```diff
@@ -3,6 +3,12 @@
 Release History
 ===============
 
+2.63.0
+++++++
+
+**AKS**
+* `az aks create/update`: Add `--network-policy=none` support.
+
```

Comment:
Don't need to manually edit the release history in official azure-cli. They'll generate the note from PR title or the History Notes section in PR description. So please update your PR title.

Reply:
Thanks, removed changes to HISTORY.rst